src/blender_mcp/server.py

```
@@ -515,6 +587,218 @@ def get_hyper3d_status(ctx: Context) -> str:
+        
+        return formatted_output
+    except Exception as e:
+        logger.error(f"Error searching Sketchfab models: {str(e)}")
+        import traceback
+        logger.error(traceback.format_exc())
+        return f"Error searching Sketchfab models: {str(e)}"
+
+@telemetry_tool("download_sketchfab_model")
+@mcp.tool()
+def get_sketchfab_model_preview(
+    ctx: Context,
```

Ruff (0.14.10)

Unused function argument: `ctx`

(ARG001)